### ISSUE TITLE
Docker plugin: Docker plugin gets enable after renaming the hpe.conf [DEFAULT] as 3PAR MU3

### ISSUE THREAD

**Issue by sandesh-desai** [hpe-storage/python-hpedockerplugin#302]
Docker plugin gets enable after renaming the hpe.conf settings [DEFAULT] as [3PAR MU3]

Testbed Details:
Host:192.168.68.31
Host OS: Ubuntu Release: 16.04
Docker Version: 17.06.2-ee-16
3PAR array: 15.212.192.252 [3PAR MU3] ,192.168.67.5 [3par1],192.168.67.2[3par2]

Steps as follows:

1) vi /etc/hpedockerplugin/hpe.conf
2) Rename [DEFAULT] to [3PAR MU3]
3) Disable plugin
  docker plugin disable hpe
4) Enable plugin 
   docker plugin enable hpe

While creating a volume it gives an error.

Attaching the hpe.conf,syslog and volume creation error
[Creating volume.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/2345436/Creating.volume.txt)

[hpe.conf settings.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/2345430/hpe.conf.settings.txt)
[syslog.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/2345432/syslog.txt)

**Comment by atotala** [comment]
DEFAULT section is mandatory as it contains many other details rather than arrays detail, which is mandatory for plugin to work properly

**Comment by wdurairaj** [comment]
[DEFAULT] section is required for the plugin to operate. So changing the DEFAULT to something will result in plugin not operating correctly. We will add this note in the documentation.

**Comment by sandesh-desai** [comment]
Can we mark this as a documentation label ?

**Comment by imran-ansari** [comment]
@prablr79 - Please review the below content:


"DEFAULT" section in hpe.conf is mandatory for the plugin to work. Not having it or renaming it would prevent the plugin from getting enabled.

**Comment by prablr79** [comment]
Thats correct, please go ahead and update the documentation in master. i can close this bug..
